sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 12.0.0b5 (Unreleased)
+* This version and all future versions will require Python 2.7 or Python 3.6+, Python 3.5 is no longer supported.
```

Comment:
you also need to update the [prerequisites section of the readme](https://github.com/Azure/azure-sdk-for-python/blob/c01f38678c41029d7ffae8fe6462ad9cc7c865b8/sdk/tables/azure-data-tables/README.md#prerequisites)